tests/storage/linstor/test_linstor_sr.py

```diff
@@ -2,31 +2,31 @@
 import pytest
 import time
 
-from .conftest import GROUP_NAME, create_linstor_sr, destroy_linstor_sr
+from .conftest import STORAGE_POOL_NAME
 from lib.commands import SSHCommandFailed
 from lib.common import wait_for, vm_image
 from tests.storage import vdi_is_open
 
 # Requirements:
 # - one XCP-ng host >= 8.2 with an additional unused disk for the SR
 # - access to XCP-ng RPM repository from the host
-# - a repo with the LINSTOR RPMs must be given using the command line param `--additional-repos`
 
 class TestLinstorSRCreateDestroy:
-    vm = None
-
-    def test_create_sr_without_linstor(self, hosts, lvm_disks):
-        master = hosts[0]
+    """
+    Tests that do not use fixtures that setup the SR or import VMs,
+    because they precisely need to test SR creation and destruction,
+    and VM import.
+    """
 
+    def test_create_sr_without_linstor(self, host, lvm_disk):
         # This test must be the first in the series in this module
-        assert not master.binary_exists('linstor'), \
+        assert not host.file_exists('/usr/lib/python2.7/site-packages/linstor/linstorapi.py'), \
```

Comment:
Is there any other file we could test whose path does not depend on the version of python?

Reply:
We mainly depend on the LINSTOR python library in our driver, I will see if I can find another file installed by this package.

Reply:
Well only python files are installed by this RPM...

Reply:
Ok

Reply:
Then maybe you can test for the presence of the rpm now that you wrote a function for this :)